run.py

```diff
@@ -56,6 +57,22 @@ def py_shell():
     shell.interact()
 
 
+def python_runtime():
+    # extract the absolute script path to alter sys.path and specific dunder variables
+    script = pathlib.Path(sys.argv[2]).expanduser().absolute()
+    sys.path.insert(0, str(script.parent))
+
+    # update passed args so they don't start with "<binary> python"
+    sys.argv = sys.argv[2:]
```

Comment:
```suggestion
    sys.argv[:] = sys.argv[2:]
```
Modify the list in place(doesn't create a new list)